nextflow.config

```diff
@@ -117,5 +119,19 @@ profiles {
         executor {
             name = "local"
         }
+        includeConfig 'config/local.config'
+    }
+    local_docker {
```

Comment:
This has some repeated bits with `local` and `local_docker`, I think we can just have `local`, `docker`, `singularity` as separate entries and then use them as needed `-profile local,docker` or `-profile local,singularity`

something like this:
```
docker {
    enabled = true
    fixOwnership = true
    autoMounts = true
}

singularity {
    enabled = true
    autoMounts = true
    remove = false
}

local {
    params.outdir = "result"
    executor {
        name = "local"
        cpus = 2
    }
    includeConfig 'config/local.config'
}
```

Reply:
I'm going to make this change in a different branch